docs/users.rst

```diff
@@ -636,6 +636,49 @@ Copy the osmosis command generated by conversion tool and run from the command l
 * Environment variables: PWD=/path/to/beam/folder
 
 
+11. Warmstart
+
+We can configure beam to run from previous state. This configuration can be controlled by following parameters in config file::
+
+ * beam.warmStart.type = "disabled"
+ * beam.warmStart.path = "https://s3.us-east-2.amazonaws.com/beam-outputs/run140-base__2018-06-26_22-20-49_28e81b6d.zip"
+ * beam.warmStart.samplePopulationIntegerFlag = 0
+ * beam.warmStart.prepareData = false
+ * beam.warmStart.skimsFilePaths = [
+                                     {
+                                         skimType = "String"
+                                         skimsFilePath = ""
+                                     }
+                                   ]
+
+
+
+We can configure each param as following
+
+- `beam.warmStart.type`: This parameter control type of warm start. We have following options for this param.
```

Comment:
```suggestion
- `beam.warmStart.type`: This parameter controls the type of warm start, with the possibilities being::
```

Reply:
Does the other colon need to be doubled?

Reply:
Applied suggestion. But not double colon. double colon have special meaning in Markdown language . Its create a block .